docs/get-started/v2/get-started/prebuilt/prebuilt.mdx

```diff
@@ -46,7 +48,7 @@ In this guide, we will create role specific prebuilt room links and share/embed
 <FlexContainer>
 <Callout icon="link" title="Prebuilt Link Structure">
 Prebuilt links are role specific room links with a format `https://<template-subdomain>.app.100ms.live/<room-code>` where,
-    - `template-subdomain`- is the subdomain that you’ve defined at the time of template creation on [100ms Dashboard](https://dashboard.100ms.live/dashboard). Copy `template-subdomain` from Template details page on 100ms Dashboard
+    - `template-subdomain`- is the subdomain that you’ve defined at the time of template creation on [100ms Dashboard](https://dashboard.100ms.live/dashboard). Copy `subdomain` from Template details page on 100ms Dashboard
```

Comment:
```suggestion
    - `template-subdomain`- is the subdomain that you’ve defined at the time of template creation on [100ms Dashboard](https://dashboard.100ms.live/dashboard). Copy the subdomain (e.g. "videoconf-app") from Template details page on 100ms Dashboard
```

Reply:
actually this is still not the best. 

dashboard helps you to copy the domain. 
need to fix this.